### ISSUE TITLE
Wireless configuration is static after installing ArchL (with Gnome in my case)

### ISSUE THREAD

**Issue by AntiBigTech** [archlinux/archinstall#264]
Dear people of Archinstall:

When I was being asked for: what configuration of the network Interface would I choose? (during the archinstall steps), I choose the first option [Save the current network configuration ~ and I was installing it whith my wifi enabled whith iwctl]...

And after the installation I noticed warious facts:

First one, I can't see the Wifi Button on the main menu of the upper bar:
![Screenshot from 2021-04-09 19-07-16](https://user-images.githubusercontent.com/82244838/114251627-531ca700-9967-11eb-989f-ea100324f24c.png)

But i wasn't asked for which Wifi networ to choose or typing the password for it. It simply "catched" my first wifi enabled, but it don't allow me to GUI search other wifi networks...

Second: It is not possible to open the network configuration from the settings window: 
![Screenshot from 2021-04-09 19-19-11](https://user-images.githubusercontent.com/82244838/114251939-890e5b00-9968-11eb-9669-2c5abdd62e5c.png)

Third: I manual installed the NetworkManager (with "$ sudo pacman -S networkmanager"), but it doesn't take effect.

It's obwious that the installer was too "Literal" whith maintaining the first config, and don't installed that service with the GUI integration with Gnome.

Thanks.

**Comment by dylanmtaylor** [comment]
> First one, I can't see the Wifi Button on the main menu of the upper bar:

GNOME doesn't display this without the networkmanager package installed and the systemd service running, do

```
sudo pacman -S networkmanager
sudo systemctl enable --now NetworkManager
```

**Comment by dylanmtaylor** [comment]
This is improved in the version in master, which will be in 2.1.4 by the way - there is a new option to use NetworkManager that takes care of this for you so you don't have to deal with it.

**Comment by dylanmtaylor** [comment]
I thought about adding networkmanager to GNOME and KDE profiles in the past because of this specific issue, but our goal in the installer is to basically bootstrap a minimal functional desktop and let users configure it without making choices for them, and not everyone wants networkmanager. (https://github.com/archlinux/archinstall/pull/245).

**Comment by AntiBigTech** [comment]
> > First one, I can't see the Wifi Button on the main menu of the upper bar:
> 
> GNOME doesn't display this without the networkmanager package installed and the systemd service running, do
> 
> ```
> sudo pacman -S networkmanager
> sudo systemctl enable --now NetworkManager
> ```

Thanks a lot! It worked perfect.

**Comment by AntiBigTech** [comment]
> This is improved in the version in master, which will be in 2.1.4 by the way - there is a new option to use NetworkManager that takes care of this for you so you don't have to deal with it.

Excellent.

**Comment by AntiBigTech** [comment]
> I thought about adding networkmanager to GNOME and KDE profiles in the past because of this specific issue, but our goal in the installer is to basically bootstrap a minimal functional desktop and let users configure it without making choices for them, and not everyone wants networkmanager. (#245).

I think that (newbey) explanations like this would be fantastic to be on this proyect wiki. Other interesting point to be -more explained- may be in the -desktop environment- seccion, because in the list appears the word -desktop- and its confusing... for example I thought that it was refering to Xfce, mate... or another -windows like- Desktop Environment.

**Comment by Torxed** [comment]
> > I thought about adding networkmanager to GNOME and KDE profiles in the past because of this specific issue, but our goal in the installer is to basically bootstrap a minimal functional desktop and let users configure it without making choices for them, and not everyone wants networkmanager. (#245).
> 
> I think that (newbey) explanations like this would be fantastic to be on this proyect wiki. Other interesting point to be -more explained- may be in the -desktop environment- seccion, because in the list appears the word -desktop- and its confusing... for example I thought that it was refering to Xfce, mate... or another -windows like- Desktop Environment.

Just a small clarification, `NetworkManager` is actually installed with Gnome as it's a hard dependency of the [gnome](https://archlinux.org/groups/x86_64/gnome/) group that we install. However, the guided installer didn't have an option to enable/manager NetworkManager from this release - but it does get installed but you would have to enable it manually as mentioned above.

I apologies for this inconvenience and as Dylan mentioned, this will be fixed in the next release. Appreciate the feedback!

**Comment by Torxed** [comment]
This was fixed in [v2.1.4](https://github.com/archlinux/archinstall/releases/tag/v2.1.4) *(released yesterday)*. We've introduced the option to use NetworkManager as the network configuration tool, which should solve this issue.

**Comment by AntiBigTech** [comment]
> This was fixed in [v2.1.4](https://github.com/archlinux/archinstall/releases/tag/v2.1.4) _(released yesterday)_. We've introduced the option to use NetworkManager as the network configuration tool, which should solve this issue.

THANKS A LOT. ¡God Bless you!
